idaes/generic_models/properties/helmholtz/helmholtz.py

```diff
@@ -475,6 +475,7 @@ def p(self, **kwargs):
 
     def v(self, **kwargs):
         blk, delta_liq, delta_vap, tau, x = self.basic_calculations(**kwargs)
+        # PYLINT-TODO check if self.mw (flagged as no-member by pylint) isn't actually self.param.mw
```

Comment:
`self.mw` should be a reference (of some form) to `self.param.mw`. However, they might show up as different objects.

Reply:
Just to be sure I understand for the purpose of addressing the pylint error: is it OK to change `self.mw` to `self.param.mw` here (which would make pylint happy without needing directives), or not?

Reply:
I note that a couple of later methods use `self.params.mw` instead of `self.mw`, so we are not even consistent internally with how we are doing this. I did check the code, and `self.mw` is declared (as part of the `build` method), so it will exist, but it seems pylint doesn't see this (probably because the current method is a peer of build).

Given some other things that could potentially crop up, I think making this `self.params.mw` is probably OK.